Grunt scripts:adaptpostbuild task is throwing an error

### Subject of the issue
The post-build.js script for the xAPI extension is not executing as part of the Grunt build process.

### Your environment
* Adapt Framework v3.2

### Steps to reproduce
Run any Grunt build task on a course with the xAPI extension enabled.

### Expected behaviour
The post-build.js script inside adapt-contrib-xapi would run.

### Actual behaviour
The following is output in the log:
```
Running "scripts:adaptpostbuild" (scripts) task
Error: Cannot find module 'src/extensions/adapt-contrib-xapi/scripts/post-build.js'
```

Thanks to @NayanKhedkar and @ambadasavhad for noticing this.

--- a/grunt/tasks/scripts.js
+++ b/grunt/tasks/scripts.js
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
                 if (!script) return done();
 
                 try {
-                    var buildModule = require(path.join(plugindir, script));
+                    var buildModule = require(path.join(process.cwd(), plugindir, script));
                     buildModule(fs, path, grunt.log.writeln, {
                        sourcedir: options.sourcedir,
                        outputdir: options.outputdir,

